src/Microsoft.ApplicationInsights.AspNetCore/Logging/Implementation/ApplicationInsightsLogger.cs

```diff
@@ -0,0 +1,70 @@
+﻿namespace Microsoft.ApplicationInsights.AspNetCore.Logging
+{
+    using System;
+    using System.Collections.Generic;
+    using Microsoft.ApplicationInsights.DataContracts;
+    using Microsoft.Extensions.Logging;
+
+    internal class ApplicationInsightsLogger : ILogger
+    {
+        private readonly string categoryName;
+        private readonly TelemetryClient telemetryClient;
+        private readonly LogLevel minimumLevel;
+
+        public ApplicationInsightsLogger(string name, TelemetryClient telemetryClient, LogLevel minimumLevel)
+        {
+            this.categoryName = name;
+            this.telemetryClient = telemetryClient;
+            this.minimumLevel = minimumLevel;
+        }
+
+        public IDisposable BeginScope<TState>(TState state)
+        {
+            return null;
```

Comment:
Will this get implemented? User-supplied scopes would be important IMO. 
However, this would also include all request-scopes from aspnet/hosting which is already covered by the DiagnosticListener so this might lead to duplicated information?! Should they be filtered out explicitly?